torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -1210,23 +1216,30 @@ class TensorExprFuser {
   size_t min_group_size_;
   // compose Runtime Type Guard and Kernel in one op
   bool add_composed_op_;
+  // generalize static shapes to dynamic shapes
+  bool fuse_to_dynamic_shapes_;
 };
 
 void FuseTensorExprs(
     std::shared_ptr<Graph>& graph,
     size_t min_group_size,
-    bool add_composed_op) {
+    bool add_composed_op,
+    bool fuse_to_dynamic_shapes) {
   GRAPH_DUMP("Before TExprFuser: ", graph);
 
   // Temporary change for Block code generation.
   if (tensorexpr::getTEGenerateBlockCode()) {
     min_group_size = 1;
   }
 
+  if (add_composed_op) {
+    TORCH_INTERNAL_ASSERT(fuse_to_dynamic_shapes, "Fusing static shapes with composed op NYI");
```

Comment:
Why is this an error?

Reply:
The `add_composed_op` path always fuses to dynamic shapes right now, we didn't implement composed op + static shapes (bc there is no use case for it)